tests/transports-governance/test_runner_hierarchical_comprehensive.py

```
@@ -0,0 +1,830 @@
+        report = runner.run_all_suites()
+
+        # Exit with appropriate code
+        if report["status"] == "passed":
+            sys.exit(0)
+        else:
+            sys.exit(1)
+
+    except KeyboardInterrupt:
+        print("\n🛑 Test execution interrupted by user")
+        sys.exit(130)
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)